packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -85,17 +106,24 @@ const GroupAction = ({
         evt: SelectionChangeEventArgs,
         selectionProvider: ISelectionProvider,
       ) => {
-        if (queryGenerationType === "Selection") {
-          const selection = selectionProvider.getSelection(evt.imodel, evt.level);
-          const query = selection.instanceKeys.size > 0 ? `SELECT ECInstanceId FROM ${selection.instanceKeys.keys().next().value}` : "";
+        if (localQueryGenerationType === "Selection") {
+          const selection = selectionProvider.getSelection(
+            evt.imodel,
+            evt.level,
+          );
+          const query =
+            selection.instanceKeys.size > 0
+              ? `SELECT ECInstanceId FROM ${selection.instanceKeys.keys().next().value
```

Comment:
Do not split the string into multiple lines. This just makes it more difficult to read. For ternary operators I usually use this formatting:
```
condition
  ? { execute when condition is true }
  : { execute when condition is false }
```

Also, the condition can be on the same line as `const query`.